db/zkdbSchema.sql

```diff
@@ -353,6 +355,8 @@ CREATE TABLE IF NOT EXISTS `tracknames` (
 
 --
 -- Table structure for table `tracks`
+-- Upgrade for existing DB:  
+-- alter table tracks add created timestamp null default NULL;
```

Comment:
Comments for upgrade for existing DB need to be removed, and the actual ALTER statement(s) moved into a .sql script that can be run to do the conversion.  See the script 'convert_v1_to_v2.sql' on the kzsu branch as a model.

Reply:
Hello Eric, this one is still open...

Reply:
comment removed. can't add the convert script because this PR is based on master, not the kzsu branch. will submit another PR for it if/when this one gets accepted.

Reply:
`master` is the base zookeeper codebase; `kzsu` contains kzsu specifics.  Your changes are against master, and so the script also.  That is to say, the script is not specific to kzsu, it is related to your changes, which are against the base, and anyone using the base wants/needs the conversion script.  So it's fine and appropriate to include here.

That said, you have raised some very important points about the code organization which we need to address first, the kzsu branch being one of them.  The kzsu branch needs to go away.  I have opened a separate PR to unify the branches.

Reply:
yes, i was painfully aware of the functionality differences between the two branches. hence i added the css classes to minimize the differences. having said that, i do think that the station's best interests might  be best served by it having its own dedicated for.

just to confirm, you would like me to add the conversion scripts to master. will you then be prepared to accept this PR?

Reply:
> yes, i was painfully aware of the functionality differences between the two branches. hence i added the css classes to minimize the differences. having said that, i do think that the station's best interests might be best served by it having its own dedicated for.

The branches have been merged, but not the code paths!  The base will never execute kzsu code.  Kzsu code paths in the base would be completely unacceptable.

The solution was obvious; I am not sure why I did not do it in the first place and avoid the kzsu branch nonsense.  Zookeeper is object oriented.  When station-specific code is required, classes can be subclassed.  The subclass inherits all functionality from the superclass except those method(s) that are overridden for the station-specific behaviour.  The factories instantiate the appropriate class (superclass or subclass) based on the configuration files.

Thus, there is no need for DIVs to hide the kzsu header, because the kzsu code path is never run on the base.

> just to confirm, you would like me to add the conversion scripts to master. will you then be prepared to accept this PR?

Thanks for committing.  Changes to the schema require conversion scripts.  (The only reason the convert_v1_to_v2 script is on the kzsu branch is because only kzsu ever had a v1 database, and thus only kzsu would ever use that script; I did not want to confuse anyone else by sticking it into master.)

The next steps involve my continuing to finesse the branch unification PR, and once I am happy with it, merge that.  This will simplify your PR, and avoids our having to deal with the kzsu branch in merging your PR.